spec.md

```diff
@@ -790,9 +790,11 @@ For success responses, the following fields are defined:
 }
 ```
 
-If the response contains `"state": "failed"` then the Platform MUST send a
+If the response contains `"state": "failed"` then the Platform SHOULD send a
```

Comment:
@mattmcneeney the `GET /v2/service_instances/:instance_id/last_operation` being common to all async requests, this sentence would then be applied on failed async updates, where it should not, per #508 

Hence the suggestion in https://github.com/openservicebrokerapi/servicebroker/pull/591#issuecomment-415676107 to only keep this sentence in the orphan section of the specs and to specialize it w.r.t. the associated async operation.

Reply:
Ah, good point. I can't work out how to word this to cover this scenario. Do you have any suggestions?

Reply:
I think remove the section here entirely, and then add it to the end of the orphan mitigation section, something like
`
If a GET /v2/serviceinstances/:service_id/last operation response contains `"state": "failed"` then the Platform SHOULD send a deprovision request to the Service Broker to prevent an orphan being created on the Service Broker if appropriate. The Platform MAY allow end users to determine when this deprovision request is sent to the Service Broker in order to allow them to investigate why the failure occurred. However, while the Platform will attempt to send a deprovision request, Service Brokers MAY automatically delete
`